src/components/VideoPlayer/Player.js

```diff
@@ -167,6 +167,11 @@ class VideoPlayer extends Component {
         loading: false
       }))
     }
+    this.onVolumeChange = () => {
+      if(globalState.muted !== this.video.muted) {
```

Comment:
Let's consistently have a space after the if:  `if (globalState`

Reply:
sorry for that one, standard usually does that for me in precommit...